examples/PCovR_Scaling.ipynb

```diff
@@ -0,0 +1,286 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "toc": true
+   },
+   "source": [
+    "<h1>Table of Contents<span class=\"tocSkip\"></span></h1>\n",
```

Comment:
I find having a TOC a bit weird in small example, why include it?

Reply:
Oh, it was autogenerated by my computer -- it won't show up in the docs or necessarily in anyone else's build of the NB.

Reply:
There is something similar in https://scikit-cosmo.readthedocs.io/en/latest/read-only-examples/Selectors%2BPipelines.py.html, is this expected to be removed?